index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <nav>
+      <a href="/"><img src="images/logo.png" alt="menu" /></a>
+      <img src="images/menu.png" alt="menu" />
+    </nav>
+    <header>
+      <h1>
+        I’m Yasin <br />
+        Glad to see <br />
+        you!
+      </h1>
+      <p>
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <a href="">LET'S CONNECT</a>
+      <div class="social-icons">
+        <img src="images/twitter.png" alt="menu" />
+        <img src="images/Linkedin.png" alt="menu" />
+        <img src="images/m.png" alt="menu" />
+        <img src="images/github.png" alt="menu" />
+        <img src="images/peace.png" alt="menu" />
```

Comment:
- [ ] Let's make sure we wrap each img with `ul` and `a` tag anchor tag will help us to navigate to each social media 
should look like this 
```html
<ul>
<a href= "twitter.com/yourusername">
<img src="images/twitter.png" alt="menu" />
</a>
</ul>
```

Reply:
Thanks for the review